README.adoc

```diff
@@ -1,50 +1,111 @@
++
+[source,bash]
+----
+user@user-pc ~/boresha/agritrader $ lsusb
+Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
+Bus 003 Device 003: ID 093a:2521 Pixart Imaging, Inc. Optical Mouse
+Bus 003 Device 002: ID 0534:0002 Motorola, Inc.
+Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
+Bus 001 Device 055: ID 04e8:6860 Samsung Electronics Co., Ltd Galaxy (MTP)
+Bus 001 Device 002: ID 046d:0a03 Logitech, Inc. Logitech USB Microphone
+Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+----
++
+The path would then be `/dev/bus/usb/001/055` for the Samsung device (phone).
+
+. Check that the Docker image can find your phone using `./dockerize.sh adb /path/to/device/`.
+The output should include a device.
++
+[source,bash]
+----
+user@user-pc ~/boresha/agritrader $ ./dockerize.sh adb /dev/bus/usb/001/055
+List of devices attached
+* daemon not running. starting it now at tcp:5037 *
+* daemon started successfully *
+ce12160cdb9b2f1905	device
+----
+
+. Build the package and deploy it to your phone using `./dockerize.sh android /path/to/device`.
+This should take a few minutes before anything shows up on your phone.
+
+. Once the app is running on your phone, enable hot reloading by shaking your phone and selecting _Enable Hot Reloading_.
+
+NOTE: The phone will need to be disconnected and reconnected for every build (`./dockerize.sh android /path/to/device`).
+
+=== Setting up a Development Environment Locally
+
+Follow steps found in the `Building Projects with Native Code` tab in the link:https://facebook.github.io/react-native/docs/getting-started.html[React Native documentation for Getting Started] for installing dependencies and setting up the mobile development environment, such as the Android SDK and build tools.
+
+==== Setting up a Device
+
+To try running the code on a device, follow the steps found in the link:https://facebook.github.io/react-native/docs/running-on-device.html[React Native documentation for Running your app on a Device].
+
+==== Common Issues
+
+* Make sure the `JAVA_HOME` and `ANDROID_HOME` environment variables are set.
+
+* Make sure the `adb` command is available from the terminal.
+The sdk platform tools directory may have to be added to the `PATH` environment variables.
+The platform tools directory can be found in `D:\Users\<your username>\AppData\Local\Android\sdk\platform-tools`.
+
+* When trying to run the application on a device, make sure a phone is connected.
+Use `adb devices` to check for a connected device.
+
+
+==== Running the Application
+
+Once a development environment has been setup, the application can be run with the following steps.
+
+. Clone the repository.
+. Run `yarn install` to install the dependencies.
+. Ensure a device is connected to the computer or an emulator is set up.
+. Run `npm run start:android` or `npm run start:ios` to run the application.
+
+==== Running Tests
```

Comment:
do we need to talk about how to run tests using docker?

Reply:
or how to use `./dockerize.sh run`?

Reply:
Note added on how to use `./dockerize.sh run`.